unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/BuilderMode/States/BuilderInWorldMode.cs

```diff
@@ -35,10 +33,12 @@ public class BuilderInWorldMode : MonoBehaviour
 
     protected List<BuilderInWorldEntityAction> actionList = new List<BuilderInWorldEntityAction>();
 
-    public virtual void Init()
+    public virtual void Init(BIWContext biwContext)
     {
-        gameObject.SetActive(false);
-        builderInWorldEntityHandler.OnEntityDeleted += OnDeleteEntity;
+        biwEntityHandler = biwContext.entityHandler;
+        biwSaveController = biwContext.saveController;
+        ibiwActionController = biwContext.actionController;
+        biwEntityHandler.OnEntityDeleted += OnDeleteEntity;
     }
```

Comment:
We should remove all the `ibiw` and `biw` field prefixes here so they are called `entityHandler`, `saveController` and `actionController`